Add `Touhou Mania Cup 3rd` tournament article (#11511)
ppy/osu-wiki: wiki/Tournaments/TMC/3rd/en.md

---
tags:
  - TMC
  - THMC
  - THMC 3rd
  - THMC3nd
---

# Touhou Project Mania Cup 3rd

![Touhou Project Mania Cup 3nd banner](img/logo.jpg)

> The **Touhou Project Mania Cup 3nd** (***THMC 3nd***) was a team-based 2v2 osu!mania 4K tournament hosted by ::{ flag=CN }:: [\[GB\]Sanae](https://osu.ppy.sh/users/11238501). The main theme of this tournament the Touhou Project. The pool used for THMC 3rd is 100% based on original and fan-made Touhou music. It is the third instalment of the Touhou Project Mania Cup.

Reviewer: ```suggestion
The **Touhou Project Mania Cup 3nd** (***THMC 3nd***) was a team-based 2v2 osu!mania 4K tournament hosted by ::{ flag=CN }:: [\[GB\]Sanae](https://osu.ppy.sh/users/11238501). The main theme of this tournament the Touhou Project. The pool used for THMC 3rd is 100% based on original and fan-made Touhou music. It was the third instalment of the Touhou Project Mania Cup.
```

    - The minimum amount of required players is considered as the amount of players needed to play a beatmap without any vacant spots on the lobby (i.e. 2 participants must be present for the match to begin).
13. If a player disconnects between beatmaps and the team cannot provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match).
14. Exchanging players between games is allowed without limitations.
15. Beatmaps must not be reused in the same match, except for rematches.
16. **The minimum size for a team is 2 players, and the maximum is 4.**
17. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied at the referee's discretion. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, and other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.

> 18. All players and staff must be treated with respect. Instructions of the referees and the Tournament Management are to be followed. Decisions labelled as final are not to be objected.

Reviewer 2: ```suggestion
18. All players and staff must be treated with respect. Instructions of the referees and the tournament management are to be followed. Decisions labelled as final are not to be objected.
```

19. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches and must be exchanged for the duration of the punishment.
20. Penalties for violating the tournament rules include, but are not limited to:
    - Exclusion of specific players for one beatmap.
    - Exclusion of specific players for an entire match.
    - Declaring the match as forfeited, or as a win by default for the other team.
    - Disqualification from the entire tournament.
    - Disqualification from the current and future official tournaments, until appealed.
21. Referees may allow, at their discretion, lower or higher tolerances for timers.

> 22. The Tournament Management reserves the right to modify these rules at any moment. Any such changes will be announced in advance.

Reviewer 2: ```suggestion
22. The tournament management reserves the right to modify these rules at any moment. Any such changes will be announced in advance.
```

### Tournament registration

> 1. Please complete the [registration form](https://forms.gle/NUEf7Mwv94UvSpzu7) to registrate.

Reviewer 2: ```suggestion
1. Please complete the [registration form](https://forms.gle/NUEf7Mwv94UvSpzu7) to register.
```

   - If any member in a team is blacklisted by organisers or considered "suspicious" by the account support team, the whole team will be disqualified. **Blacklists in other tournaments will be considered as well.**
   - There are no rank requirements for this tournament, but player must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
5. All successfully formed teams will be published after the registration phase.
   - A team should have at least 2 players registered to have the chance to play in the Qualifiers.
   - The top 32 seeded teams will qualify to the Round of 32.
6. Tournament staff members are **not** allowed to play in the tournament, with the exception of the commentators.
7. All information of the tournament will be ONLY available on the [Discord server](https://discord.gg/cAr5VHG7WX). All captains are asked to join the server in case of any issues.

### Qualifier instructions

1. During Qualifiers, all teams will play a specific pool designed by the mappool selectors.

> 2. The qualifier pool contains 5 maps, all of which will use FreeMod rules. There will be no tiebreakers for this stage.

Reviewer 2: ```suggestion
2. The qualifier pool contains 5 maps, all of which will use Free Mod rules. There will be no tiebreakers for this stage.
```

3. The qualifier pool will follow this structure:
   1. Stage 1: Hybrid
   2. Stage 2: Rice Jack
   3. Stage 3: Rice Mixed
   4. Stage 4: LN Mixed
   5. Stage 5: Rice Stamina
   6. Stage 6: LN Density
4. Teams will have to play the mappool twice at a designated time. The team's best playthrough will be used for seeding.
   - Teams may skip individual maps or the second run altogether if they so desire.
5. The mappool will be played according to the order listed above.
6. Each team must have 2 players for each map. They can be exchanged freely after a map is concluded.
7. An optional 5-minute break will be offered between the first and second playthrough of the mappool.
8. All teams will play their qualifier matches in separate rooms. It is suggested that teams do not broadcast or share their results publicly to avoid seed manipulation.
9. Only the top 32 seeded teams will advance to the Round of 32.

### Stage instructions

1. Following the Qualifiers, a double-elimination bracket will be played. This means that the winning team moves to the next stage while the loser gets moved to the losers' bracket.

> 2. Teams in the losers bracket must play 2 matches each weekend starting from the Quarterfinals.

Reviewer 2: ```suggestion
2. Teams in the losers' bracket must play 2 matches each weekend starting from the Quarterfinals.
```

   - Team captains will be notified of their match via ping on Discord by the referee of their match 15 minutes before the match is scheduled to commence.
   - The room settings are: Game mode: "osu!mania"; Team mode: "Team Vs"; and Score mode: "ScoreV2". The room name must follow this pattern: `THMC3 ({Stage}): ({Blue Team}) vs ({Red Team})`. The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
2. Each captain can ban **one beatmap** from the pool. These beatmaps can not be picked by any team for the entire duration of the match.
3. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
4. Each captain must use `!roll` once in `#multiplayer`.
   - The winner of the `!roll` will select which team will pick first.
   - The loser of the `!roll` will select which team will ban first.
5. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a team takes more time than allotted, the procedure adopted will be as follows:
   - For the first occurrence:
     - The team will receive a verbal warning from the referee
   - On subsequent occurrences:

>      - For a pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the Tiebreaker.

Reviewer 2: ```suggestion
     - For a pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the tiebreaker.
```

     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.

>    - Repeat offenders may receive further sanctions from the Tournament Management.

Reviewer 2: ```suggestion
   - Repeat offenders may receive further sanctions from the tournament management.
```

>    - Each team will receive one "tactical timeout" of two minutes, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional*.

Reviewer 2: when bolding, italicising or linking a whole sentence, the terminating punctuation goes on the inside

```suggestion
   - Each team will receive one "tactical timeout" of two minutes, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
```

     - This timeout is autimatically consumed when regular picking time runs out.

>      - The tactical timeout may be called on a tiebreaker, provided that both team's captains agree to it.

Reviewer 2: ```suggestion
     - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.
```

   - Each team will receive one technical timeout of 5 minutes, to be used only when a technical issue occurs.
6. Results for the Qualifiers will be published via a statistics sheet.

### Mappool instructions

1. There will be an independent mappool for each stage.

> 2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.

Reviewer 2: ```suggestion
2. Each mappool consists of a fixed amount of maps each stage which will all be played under Free Mod conditions. This means that there is a unique Free Mod bracket.
```

3. The mappool sizes are as follows:

>    - Qualifiers: 6 beatmaps
>    - Round of 32 and Round of 16: 11 beatmaps
>      - Containing: 4 Rice maps, 2 LN maps, 2 Hybrid map, 1 Tiebreaker.
>    - Quarterfinals and Semifinals: 13 beatmaps
>      - Containing: 5 Rice maps, 3 LN maps, 2 Hybrid maps, 1 Tiebreaker.
>    - Finals: 13 beatmaps
>      - Containing: 6 Rice maps, 3 LN maps, 3 Hybrid maps, 1 Tiebreaker.
>    - Grand Finals: 15 beatmaps
>      - Containing: 7 Rice maps, 4 LN maps, 3 Hybrid maps, 1 Tiebreaker.

Reviewer 2: ```suggestion

| Stage | Beatmaps | Types |
| :-: | :-- | :-- |
| Qualifiers | 6 |  |
| Round of 32 & Round of 16 | 11 | 4 Rice, 2 LN, 2 Hybrid, 1 Tiebreaker |
| Quarterfinals & Semifinals | 13 | 5 Rice, 3 LN, 2 Hybrid, 1 Tiebreaker |
| Finals | 13 | 6 Rice, 3 LN, 3 Hybrid, 1 Tiebreaker |
| Grand Finals | 15 beatmaps | 7 Rice, 4 LN, 3 Hybrid, 1 Tiebreaker |

```

4. Each mappool has one tiebreaker, except for the Qualifiers.

> 5. Possible mod choices for all maps are [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [FadeIn](/wiki/Gameplay/Game_modifier/Fade_In), [Flashlight](/wiki/Gameplay/Game_modifier/Flashlight), and [Mirror](/wiki/Gameplay/Game_modifier/Mirror).

Reviewer 2: ```suggestion
5. Possible mod choices for all maps are [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Fade In](/wiki/Gameplay/Game_modifier/Fade_In), [Flashlight](/wiki/Gameplay/Game_modifier/Flashlight), and [Mirror](/wiki/Gameplay/Game_modifier/Mirror).
```

> 6. The tiebreaker will be played under FreeMod conditions.

Reviewer 2: ```suggestion
6. The tiebreaker will be played under Free Mod conditions.
```

### Scheduling instructions

1. Each stage will be held on **a single weekend**.
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament staff. Each team is free to choose a time slot in which they want to play.
3. All bracket stages will be held between Saturday 0:00 and Sunday 23:59 (UTC).

> 4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the stage. The Tournament Management will try to create the schedule to respect the participants' time zones.

Reviewer 2: ```suggestion
4. Scheduling will be handled by the tournament management. Schedules will be released on the Sunday before the first matches of the stage. The tournament management will try to create the schedule to respect the participants' time zones.
```

>    - In the Semifinals and subsequent stages, team captains may inform the Tournament Management if they expect a specific time slot to be unavailable in the following week. Schedule updates will be granted to the best of the Tournament Management's ability.

Reviewer 2: ```suggestion
   - In the Semifinals and subsequent stages, team captains may inform the tournament management if they expect a specific time slot to be unavailable in the following week. Schedule updates will be granted to the best of the tournament management's ability.
```

5. **Reschedules will only be considered if both teams agree to a time.**
   - Reschedules may only be requested by a team captain.
   - Matches may not be scheduled to any time beyond **Sunday 23:59 (UTC)** of the week they are to be played at.
   - If a team needs to reschedule, the captain may make their demand **at least 24 hours** (48 hours recommended) before their match in the reschedule channel.

>    - **Do not ask for a reschedule unless it is absolutely needed. The Tournament Management reserves the right to deny any rescheduling request.**
> 6. Captains are responsible for their teams' availability. Any mishaps resulting from a lack of transparency from a team are to be resolved internally, the Tournament Management will not intervene.

Reviewer 2: ```suggestion
   - **Do not ask for a reschedule unless it is absolutely needed. The tournament management reserves the right to deny any rescheduling request.**
6. Captains are responsible for their teams' availability. Any mishaps resulting from a lack of transparency from a team are to be resolved internally, the tournament management will not intervene.
```
